pages/docs/tutorials/native/mapping-primitive-data-types-from-c.md

```diff
@@ -108,16 +110,17 @@ as it is usually an 8-bit unsigned value.
 | C | Kotlin |
 |---|--------|
 | char  |  kotlin.Byte |
+| unsigned char  |  kotlin.UByte |
 | short |  kotlin.Short |
+| unsigned short |  kotlin.UShort |
 | int   |  kotlin.Int |
+| unsigned int   |  kotlin.UInt |
 | long  |  kotlin.Long |
+| unsigned long  |  kotlin.ULong |
```

Comment:
C `long` type can be either 32-bit or 64-bit depending on the platform, so it is mapped to Kotlin `Int` or `Long`.
However `long long` type is (usually) mapped to Kotlin `Long`.

Reply:
Got it. Thanks. Fixed it as well